src/main/java/org/folio/cache/VertxCache.java

```diff
@@ -2,6 +2,9 @@
 
 import java.time.LocalDateTime;
 import java.time.temporal.ChronoUnit;
+import java.util.concurrent.CompletableFuture;
+
+import org.glassfish.jersey.internal.util.Producer;
```

Comment:
Please do not use classes that are not meant to be public libraries. In this case Producer belongs to JAX-RS Reference Implementation which is not an API.
If you need something similar and cannot find it in a public API library create your own version